diff --git a/src/modules/AdminModule.cpp b/src/modules/AdminModule.cpp
index 7906b410..c202e183 100644
--- a/src/modules/AdminModule.cpp
+++ b/src/modules/AdminModule.cpp
@@ -605,7 +605,8 @@ void AdminModule::handleSetConfig(const meshtastic_Config &c)
                 config.lora.ignore_mqtt = true; // Ignore MQTT by default if region has a duty cycle limit
             }
             if (strcmp(moduleConfig.mqtt.root, default_mqtt_root) == 0) {
-                sprintf(moduleConfig.mqtt.root, "%s/%s", default_mqtt_root, myRegion->name);
+                const char *ownerId = owner.id + 1;
+                sprintf(moduleConfig.mqtt.root, "%s/%s/%s", default_mqtt_root, myRegion->name, ownerId);
                 changes = SEGMENT_CONFIG | SEGMENT_MODULECONFIG;
             }
         }
