diff --git a/platformio.ini b/platformio.ini
index e6735ec2..33456365 100644
--- a/platformio.ini
+++ b/platformio.ini
@@ -45,7 +45,7 @@ build_flags = -Wno-missing-field-initializers
 	-DRADIOLIB_EXCLUDE_APRS=1
 	-DRADIOLIB_EXCLUDE_LORAWAN=1
 	-DMESHTASTIC_EXCLUDE_DROPZONE=1
-	-DMESHTASTIC_EXCLUDE_REMOTEHARDWARE=1
+	-DMESHTASTIC_EXCLUDE_REMOTEHARDWARE=0
 	-DMESHTASTIC_EXCLUDE_HEALTH_TELEMETRY=1
   	-DMESHTASTIC_EXCLUDE_POWERSTRESS=1 ; exclude power stress test module from main firmware
         #-DBUILD_EPOCH=$UNIX_TIME
