diff --git a/src/gps/GPS.cpp b/src/gps/GPS.cpp
index 142241c43..f5243ab7c 100644
--- a/src/gps/GPS.cpp
+++ b/src/gps/GPS.cpp
@@ -1418,7 +1418,14 @@ GPS *GPS::createGps()
 
     GpioVirtPin *virtPin = new GpioVirtPin();
     new_gps->enablePin = virtPin; // Always at least populate a virtual pin
-    if (_en_gpio) {  
+    #if defined(MESHLINK)
+    bool overrideGPSEnPin = true;
+    #else
+    bool overrideGPSEnPin = false;
+    #endif   
+    if (_en_gpio || overrideGPSEnPin) {
         GpioPin *p = new GpioHwPin(_en_gpio);
 
         if (!GPS_EN_ACTIVE) { // Need to invert the pin before hardware
diff --git a/src/main.cpp b/src/main.cpp
index 7a11ca2e0..16a360353 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -282,8 +282,10 @@ static int32_t ledBlinker()
 {
     // Still set up the blinking (heartbeat) interval but skip code path below, so LED will blink if
     // config.device.led_heartbeat_disabled is changed
+    #ifndef MESHLINK
     if (config.device.led_heartbeat_disabled)
         return 1000;
+    #endif
 
     static bool ledOn;
     ledOn ^= 1;
