diff --git a/src/gps/GPS.cpp b/src/gps/GPS.cpp
index 142241c43..d8b862361 100644
--- a/src/gps/GPS.cpp
+++ b/src/gps/GPS.cpp
@@ -1418,7 +1418,12 @@ GPS *GPS::createGps()
 
     GpioVirtPin *virtPin = new GpioVirtPin();
     new_gps->enablePin = virtPin; // Always at least populate a virtual pin
-    if (_en_gpio) {
+    #if defined(MESHLINK)
+    bool overrideGPSEnPin = true;
+    #else
+    bool overrideGPSEnPin = false;
+    #endif
+    if (_en_gpio || overrideGPSEnPin) {
         GpioPin *p = new GpioHwPin(_en_gpio);
 
         if (!GPS_EN_ACTIVE) { // Need to invert the pin before hardware
