diff --git a/src/modules/StatusLEDModule.cpp b/src/modules/StatusLEDModule.cpp
index 353317d25..5d06cc08e 100644
--- a/src/modules/StatusLEDModule.cpp
+++ b/src/modules/StatusLEDModule.cpp
@@ -182,9 +182,12 @@ int32_t StatusLEDModule::runOnce()
 #endif
 
     // Override if disabled in config
+#ifndef MESHLINK
     if (config.device.led_heartbeat_disabled) {
         CHARGE_LED_state = LED_STATE_OFF;
     }
+#endif
+
 #ifdef Battery_LED_1
     bool chargeIndicatorLED1 = LED_STATE_OFF;
     bool chargeIndicatorLED2 = LED_STATE_OFF;
