diff --git a/src/modules/StatusLEDModule.cpp b/src/modules/StatusLEDModule.cpp
index 7acf750c2..edf2de49c 100644
--- a/src/modules/StatusLEDModule.cpp
+++ b/src/modules/StatusLEDModule.cpp
@@ -124,9 +124,11 @@ int32_t StatusLEDModule::runOnce()
     }
 
     // Override if disabled in config
+    #ifndef MESHLINK
     if (config.device.led_heartbeat_disabled) {
         CHARGE_LED_state = LED_STATE_OFF;
     }
+    #endif
 #ifdef Battery_LED_1
     bool chargeIndicatorLED1 = LED_STATE_OFF;
     bool chargeIndicatorLED2 = LED_STATE_OFF;
